src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -907,6 +907,65 @@ public override IOperation VisitWhileLoop(IWhileLoopOperation operation, int? ca
             return null;
         }
 
+        public override IOperation VisitVariableDeclarationGroup(IVariableDeclarationGroupOperation operation, int? captureIdForResult)
```

Comment:
>VisitVariableDeclarationGroup [](start = 35, length = 29)

Please make necessary changes to ControlFlowGraphVerifier.CanBeInControlFlowGraph method to make sure that nodes that you've handled are in the right set. It is fine that there still can be scenarios that we do not handle, but as long as all current tests are passing, it would be good to accurately reflect final expectations in that function.

Reply:
Updated.

---
In reply to: [166806899](https://github.com/dotnet/roslyn/pull/24692#discussion_r166806899) [](ancestors = 166806899)